### PR TITLE
Add a 'grabbable' option to the new-model dialog

### PR DESCRIPTION
**Pull request by huffman**
https://highfidelity.manuscript.com/f/cases/4926/Revert-Grabbable-setting-for-imported-models-to-disabled-or-let-customers-to-define-default-setting-prior-to-importing-objects

Test plan:
- When you import a model in the edit tools, there should be a checkbox that lets you set grabbable or not... that checkbox should properly set the grabbableness of things

### CODE REVIEW

File: interface/resources/qml/hifi/tablet/NewModelDialog.qml [hunk 1]
```diff
@@ -219,7 +224,8 @@ Rectangle {
                                 params: {
                                     textInput: modelURL.text,
                                     checkBox: dynamic.checked,
```
Comment (dback2) [inline interface/resources/qml/hifi/tablet/NewModelDialog.qml]: Can we rename these params to match their variables?

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14209/HighFidelity-Beta-PR14245-f1258e4.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14209/HighFidelity-Beta-PR14245-f1258e4.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14209/HighFidelity-Beta-Interface-PR14245-f1258e4.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14209/HighFidelity-Beta-PR14245-f1258e4.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14209/HighFidelity-Beta-Interface-PR14245-f1258e4.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14221/HighFidelity-Beta-PR14245-9d13902.apk

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14221/HighFidelity-Beta-Interface-PR14245-9d13902.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14221/HighFidelity-Beta-PR14245-9d13902.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14221/HighFidelity-Beta-Interface-PR14245-9d13902.exe

**Comment by qlihifitest** [comment]
Currently Testing

**Pull request merged**